ingest/validation/validate_metadata.py

```diff
@@ -1247,13 +1253,107 @@ def review_metadata_names(metadata):
             metadata.store_validation_issue("error", "metadata_name", error_msg)
 
 
+def identify_multiply_assigned(list):
+    """ Given a list of ontologyIDs and their purported labels,
+        return list of unique multiply-assigned labels
+    """
+    ontology_tracker = defaultdict(lambda: defaultdict(int))
+    multiply_assigned = []
+    for element in list:
+        id, label = element
+        ontology_tracker[label][id] += 1
+    for label in ontology_tracker:
+        if len(ontology_tracker[label].keys()) > 1:
+            multiply_assigned.append(label)
+    multiply_assigned.sort()
+    return multiply_assigned
+
+
+def assess_ontology_ids(ids, property_name, metadata):
+    """
+    Check ordered collection of ontology IDs for increasing numeric values
+    """
+    evidence_of_excel_drag = False
+    evidence_of_excel_drag_threshold = 25
```

Comment:
Do we need to set this at 25?  My thought was that unless the number of unique IDs and corresponding labels is the same, then likely this error happened.  But there may be some subtlety that I'm not considering that is the reason for this threshold.

Reply:
For optional metadata, we might have no labels to easily detect the error.
For something like "race", a stretch of consecutive ontology IDs like NCIT_C43874, NCIT_C43875 etc are all valid IDs and all European races that could conceivably be adjacent and not due to Excel drag. Without labels, I wanted to set a threshold that would be "safe" (I admit, 10 is probably sufficient). Excel drag will typically affect a swath of at least hundreds of cells so 25 seemed like a reasonable number.